mock/mockSyfosoknad.js

```diff
@@ -110,6 +146,10 @@ function mockSyfosoknadLokalt(server) {
                 _soknad.sporsmal = sporsmal;
                 _soknad.innsendtDato = new Date().toJSON()
                     .substr(0, 10);
+                _soknad.sendtTilArbeidsgiverDato = new Date().toJSON()
```

Comment:
Skal du sette disse datoene ved gjenåpning?

Reply:
Bør jeg ikke sette dem sånn at det står at de er sendt når de har blitt ettersendt..? Er det ikke rart om jeg ikke oppdaterer dem..?

Reply:
Men du har skrevet koden under endepunktet for å gjenåpne en søknad (dvs sette status til NY når status er AVBRUTT)...

Reply:
Fikset :)